### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by christianonoh**
# Adding React Application with React Router and Components
This pull request adds a new React application that includes React Router V6 and a set of reusable components to display a list of books and create new ones.

## Changes Made
- Created a new React application using Create React App
- Created a /components directory for reusable components
- Installed React Router V6 and added it to the application
- Set up two routes for the application: "/" and "/categories"
- Created Page components for each route
- Created components inside the /components directory:
  - Book component to display individual book state and delete button
  - BookList component to render individual books
  - InputForm component to create new books
  - Navigation component to link to the two routes created
  
## Thank you for your review 🎈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/christianonoh/math-wizardry/issues).
```
Comment (anagudelogu) [inline README.md]: - [x] You did a great job working on your project documentation! 💯 However, please make sure that this link matches the correct project! 🤓 🔧

File: src/components/BookList.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      id: 2,
+    },
+  ]);
+  return (
+    <ul className="booklist-container">
+      <h1>Books</h1>
+      {
+        bookdata.map((book) => <Book key={book.id} book={book} />)
+      }
+    </ul>
```
Comment (anagudelogu) [inline src/components/BookList.jsx]: - [x] lists, in this case, `<ul>`, should only have `<li>` elements as children. Currently, you have an `<h1>` tag inside which is not valid markup. please consider moving it out! 🤓 🔧

File: src/components/BookList.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars */
```
Comment (anagudelogu) [inline src/components/BookList.jsx]: - [x] Please avoid disabling linter rules as they serve as a way of having a consistent code format and ensuring best practices! 🤓 🔧

File: src/components/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const form = () => (
+  <div>
+    Hi
+  </div>
+);
+
+export default form;
```
Comment (anagudelogu) [inline src/components/Form.jsx]: - [x] I think that you're not using this component at all, so please consider removing it as unused code can pollute your code base and make it harder to read! 🤓 🔧

File: src/components/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    { path: '/', text: 'Home' },
+    { path: 'category', text: 'Category' },
+  ]);
+  return (
+    <nav>
+      <Link to="/">
+        <img src={logo} id="logo" alt="Logo" />
+      </Link>
+      <ul className="nav-items">
+        {links.map((link) => <li key={link.text} className="nav-item"><NavLink to={link.path}>{link.text}</NavLink></li>)}
```
Comment (anagudelogu) [inline src/components/Navbar.jsx:16]: - [x] **[OPTIONAL]** Great job using a links array to handle your navigation links, this is a highly scalable way of structuring your code for possible future changes, kudos for this! 🚀 However, as this is not intended to be modified within your code, I would suggest not using `useState` for this and defining a constant instead, like this: 🤓 🔧 

```js
const LINKS = [{ path: '/', text: 'Home' }, { path: 'category', text: 'Category' }]
```

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @christianonoh, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by christianonoh** [comment]
@anagudelogu,
I wanted to take a moment to express my gratitude for your insightful review of my work. Your attention to detail and thoughtful feedback have helped me to improve my project in ways I wouldn't have been able to do on my own.
Thank you.

**Review by shella12 — APPROVED** [review]
Hi @christianonoh, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Clean code ✔️ 
- Great communication with CR 👍 
- Implemented all required changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**